accessible_api/AccessibilityRequestHandler.py

```diff
@@ -0,0 +1,124 @@
+""" Service requests for accessible objects """
+
+import sys
+from BaseHTTPServer import BaseHTTPRequestHandler
+from re import search
+from urlparse import urlsplit, parse_qsl
+from json import dumps
+from comtypes import CoInitialize
+from accessible_lib.scripts.accessible import accessible
+from accessible_lib.scripts.event import event
+from accessible_lib.scripts.commands import execute_command
+
+
+class AccessibilityRequestHandler(BaseHTTPRequestHandler):
+    def _successful_response(self, json):
+        self.send_response(200)
+        self.send_header('Content-Type', 'application/json')
+        self.end_headers()
+        self.wfile.write(json)
+
+    def _bad_response(self, message):
+        self.send_response(400, message)
+        self.send_header('Content-Type', 'application/json')
+        self.end_headers()
+
+    def _set_identifiers(self, params):
```

Comment:
nit: this is more like parse not set, so we should rename it to _parse_identifiers or _parse_params...